### ISSUE TITLE
npm test failed for ReactTestRenderer-test.js

### ISSUE THREAD

**Issue by pinakdas163** [facebook/react#9025]
**Do you want to request a *feature* or report a *bug*?**

I want to request a bug

**What is the current behavior?**
After cloning React and installing dependencies, when I ran npm test. I found one test suites to failed out 126. 

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net or similar (template: https://jsfiddle.net/reactjs/69z2wepo/).**

run command: npm test

**What is the expected behavior?**

FAIL src/renderers/testing/__tests__/ReactTestRenderer-test.js
Test suite failed to run
Cannot find module 'pretty-format' from 'ReactTestRenderer-test.js'

**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**
v15.4.2

**Comment by jddxf** [comment]
Seems that you didn't have `pretty-format` installed.Can you see it in your `node_modules` directory?

**Comment by pinakdas163** [comment]
I checked `pretty-format` module is not installed in the `node_modules` directory. I tried to install it, but I get this error. Any suggestion how to fix it?

`npm install pretty-format --save
npm ERR! Linux 4.4.0-62-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "pretty-format" "--save"
npm ERR! node v4.7.3
npm ERR! npm  v2.15.11
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package babel-eslint@7.1.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer eslint-config-fbjs@1.1.1 wants babel-eslint@^6.1.2
`
